.travis.yml

```diff
@@ -1,5 +1,5 @@
 # File managed by pluginsync
-sudo: false
+sudo: true
```

Comment:
yes, because her build failed. that's why I asked her to change it. I'll change it in the next one when I rebase Marea's PRs.